README.md

```diff
@@ -6,8 +6,6 @@
 ![](https://img.shields.io/badge/go-%3E%3D1.14.0-blue.svg)
 [![CircleCI](https://circleci.com/gh/testground/testground.svg?style=svg)](https://circleci.com/gh/testground/testground)
 
-[**Go to full docs site >>>**](https://docs.testground.ai)
```

Comment:
We have at least 2 more links 20cm down the page, linking to the Documentation and the docs website.